server/src/main/java/org/apache/uniffle/server/LocalStorageChecker.java

```diff
@@ -147,6 +150,32 @@ long getUsedSpace(File file) {
     return file.getTotalSpace() - file.getUsableSpace();
   }
 
+  protected static long getUniffleUsedSpace(File storageDir) {
```

Comment:
Could we have a better name?

Reply:
How about `getServerUsedSpace` ? Do you mean the metric also should have a better name?

Reply:
More reasonable. What's the difference between `usedSpace` and `ServerUsedSpace`?

Reply:
`usedSpace` reflect the whole disk used size instead of uniffle used, especially for colocation with Nodemanager

Reply:
I want to get the current used size of whole uniffle cluster from the metric. But this is hard for me currently.

Reply:
diskUsedSpace?

Reply:
> diskUsedSpace?

`wholeDiskUsedSpace` maybe better.

Reply:
Done